molgenis-one-click-importer/src/main/java/org/molgenis/oneclickimporter/service/Impl/EntityServiceImpl.java

```diff
@@ -64,18 +61,28 @@ public EntityServiceImpl(DefaultPackage defaultPackage, EntityTypeFactory entity
 		this.entityManager = requireNonNull(entityManager);
 		this.attributeTypeService = requireNonNull(attributeTypeService);
 		this.oneClickImporterService = requireNonNull(oneClickImporterService);
+		this.packageFactory = requireNonNull(packageFactory);
 	}
 
 	@Override
-	public EntityType createEntityType(DataCollection dataCollection)
+	public EntityType createEntityType(DataCollection dataCollection, String packageName)
```

Comment:
add java doc explaining the packageName is used for finding the package and if it is not found it is created. method name should be reflect this. Maybe this should be split up into separate fucntions